app/src/main/java/com/dgaf/happyhour/Model/UserModel.java

```diff
@@ -0,0 +1,25 @@
+package com.dgaf.happyhour.Model;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by Adam on 8/13/2015.
+ */
+@ParseClassName("User")
+public class UserModel extends ParseObject {
+    public boolean isDealUpVoted(String dealId) {
+        List<String> upVotes = getList("upVotes");
+        Collections.sort(upVotes);
+        return Collections.binarySearch(upVotes, dealId) > 0;
+    }
+
```

Comment:
Is there an easier way to "getList".  For example, passing a `public static final int DOWN_VOTES` rather than the string `downVotes`